core/src/main/java/dk/alexandra/fresco/lib/field/integer/BasicNumericContext.java

```diff
@@ -21,13 +22,15 @@
    * @param myId my party id
    * @param noOfParties number of parties in computation
    * @param fieldDefinition the field definition used in the application
+   * @param precision the fixed point precision when using the fixed point library
    */
   public BasicNumericContext(int maxBitLength, int myId, int noOfParties,
-      FieldDefinition fieldDefinition) {
+      FieldDefinition fieldDefinition, int precision) {
     this.maxBitLength = maxBitLength;
     this.myId = myId;
     this.noOfParties = noOfParties;
     this.fieldDefinition = fieldDefinition;
+    this.precision = precision;
```

Comment:
how can precision be defined here - without the F.P. library residing together? Most likely similar to maxBitLength?

Reply:
That's true. It's to avoid having to specify the precision every time a FixedNumeric is created. The only solution I can come up with is to put a default precision in DefaultFixedNumeric.

Reply:
Maybe call it that? defaultPrecision?

Reply:
Do you mean we should keep it in BasicNumericContext and just rename it, or put a constant somewhere in the fixed-lib?

Reply:
Well, I would prefer to keep the magic constants close to their use, however if it too complicated, then rename precision to defaultFixedPointPrecision

Reply:
I think that's a descent solution for now. It breaks the separation of modules but makes it easier to understand what the number actually means